app/javascript/application.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function App() {
+  return (<h1>Hello World!</h1>);
+}
+
+ReactDOM.render(
+  <App/>,
+  document.getElementById('root'),
+);
```

Comment:
- [ ] After fixing the routes I can only see the below text on the screen. The `Greeting` component is not rendering. Please make sure to fix this issue and users should be able to see the random greetings on the screen. (To fix the issue you should check the React configurations and the component calls).

![image](https://github.com/mumbereh/Hello_Webpack/assets/83803414/0d8b5afd-01da-4d7c-ade6-53efd1843d92)